### ISSUE TITLE
Team Foundation Server (TFS) Support

### ISSUE THREAD

**Issue by Bardin08** [ForNeVeR/Todosaurus#37]
Thank you for the great plugin!

It would be wonderful if your plugin supported the creation of issues in Team Foundation Server (TFS).

**Comment by ForNeVeR** [comment]
Yes, this is a direction I would like the plugin to take: extend to support more issue trackers.
